context/app/static/js/components/Detail/SummaryData/SummaryData.spec.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { render } from 'test-utils/functions';
+import SummaryData from './SummaryData';
+
+test('json button exists and has href', () => {
+  const { getByRole } = render(
+    <SummaryData entity_type="Fake" uuid="fakeuuid" status="QA" mapped_data_access_level="Public" />,
+  );
+  expect(getByRole('link')).toBeInTheDocument();
+  expect(getByRole('link')).not.toBeEmptyDOMElement();
+  expect(getByRole('link')).toHaveAttribute('href', `/browse/fake/fakeuuid.json`);
```

Comment:
```suggestion
  expect(getByRole('link')).toHaveAttribute('href', `/browse/fake/fakeuuid.json`);
```
Would the error on failure be too cryptic without these?